scripts/stop_prod_docker_environment.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Stopping docker environment"
+docker compose -f docker-compose.yml -f docker-compose-prod.yml down
```

Comment:
I don't think this is correct, because by passing it to the down method we are indirectly saying that the down method has to access some env variables during the stop phase. So this may create some misunderstanding